### ISSUE TITLE
False warning RCS1012

### ISSUE THREAD

**Issue by NN---** [JosefPihrt/Roslynator#747]
**Product and Version Used**: 
3.0.0

**Steps to Reproduce**:
editorconfig:
```
dotnet_diagnostic.rcs1012.severity=warning
```

a.cs
```cs
var a = new Dictionary<string, string> { { "", "" } };
```
